services/ui-src/src/components/reports/ModalDrawerReportPage.tsx

```diff
@@ -332,7 +332,7 @@ const entityTable = (
   return (
     <Table sx={sx.table} content={tableHeaders()}>
       {entities.map((entity: EntityShape) => (
-        <ResponsiveEntityRow
+        <EntityRow
```

Comment:
this change will be unnecessary if https://github.com/Enterprise-CMCS/macpro-mdct-mcr/pull/12378 gets merged